_layouts/agenda.html

```diff
@@ -4,12 +4,20 @@
 <div class="post">
   <header class="post-header">
     <h1 class="post-title">Agenda for {{ page.date | date: "%B %-d, %Y" }}</h1>
+    <h6 class="masthead">BAKERSFIELD COLLEGE</h6>
+    <h6 class="masthead">Computer Science Club</h6>
+    <h6 class="masthead">1801 Panorama Drive, Business Building, Room 5</h6>
+    <h6 class="masthead">Bakersfield, California 93305</h6>
```

Comment:
"preemptive delivery" - I'm hoping to figure all of this stuff out eventually so that it can become a teaching/learning opportunity. By adding a class, I can apply some kind of styling at some time. I should go back in and probably take the all caps "BAKERSFIELD COLLEGE" out